ner_v2/detectors/temporal/time/standard_time_regex.py

```diff
@@ -269,3 +287,10 @@ def _detect_time_from_standard_regex(self):
             self._update_processed_text(original_list)
 
         return time_list, original_list
+
+
+class TimeDetector(BaseRegexTime):
+    def __init__(self, entity_name, data_directory_path, timezone='UTC'):
```

Comment:
This should also accept `range_enabled` and `form_check` arguments as `en` detector accepts them